client/src/services/auth.js

```diff
@@ -29,6 +29,7 @@ class Auth {
   removeToken() {
     this.token = '';
     localStorage.removeItem(this.tokenKey);
+    localStorage.removeItem('userID');
```

Comment:
If you update this method, you should change there name. It remove not only token now

Reply:
Not understood) 
when user logs in, local storage saves token and userId. Then user logs out, and local deletes token, and it needs to delete id..

Reply:
You can rename method removeToken -> clearStorage

Reply:
@vitalii-mak if insted this line, i'll put localStorage.clear();
i think there would be no need to rename. yes? if i rename to clearStorage, it's quite not understandable I think